rbac-user-roles.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: localhost
+  vars_files:
+  - user-config.yml
+  tasks:
+    - name: add master host to group 'source_cluster'
+      add_host:
+        hostname: "{{ master_hostname }}"
+        groups: source_cluster
+        ansible_user: "{{ master_user }}"
+
+- hosts: source_cluster
```

Comment:
@fbladilo do you think the running of the pre-req should be conditional, so we control via a variable if it runs or does not?  or...do you think we should always run the pre-req?